x-pack/plugins/cases/public/common/mock/test_providers.tsx

```diff
@@ -91,19 +90,66 @@ export const testQueryClient = new QueryClient({
   },
 });
 
-export const buildCasesPermissions = (overrides: Partial<CasesPermissions> = {}) => {
+export const allCasesPermissions = () => buildCasesPermissions();
+export const noCasesPermissions = () =>
+  buildCasesPermissions({ read: false, create: false, update: false, delete: false, push: false });
+export const readCasesPermissions = () =>
+  buildCasesPermissions({ read: true, create: false, update: false, delete: false, push: false });
+export const noCreateCasesPermissions = () => buildCasesPermissions({ create: false });
+export const noUpdateCasesPermissions = () => buildCasesPermissions({ update: false });
+export const noPushCasesPermissions = () => buildCasesPermissions({ push: false });
+export const noDeleteCasesPermissions = () => buildCasesPermissions({ delete: false });
+export const writeCasesPermissions = () => buildCasesPermissions({ read: false });
```

Comment:
nit: I am a bit confused by this. Is it to test writing permission without reading permission?